projects/spring-retry/PatternMatcher_match_Fuzzer.java

```diff
@@ -0,0 +1,29 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
```

Comment:
Forgot to mention: Can you add a license header here?

Reply:
Ok, done!